### DISCUSSION TITLE
Сохранение состояний диалога.

### DISCUSSION THREAD

**Discussion by VKohai** [vknet/vk#1549] [Q&A]
Привет. Если Вы делали бота по типу опросника, где бот задаёт первый вопрос, а пользователь отвечает на него, затем бот задаёт следующий вопрос и т.д., то объясните, пожалуйста, как реализовать эту сборку данных и самое главное сохранение состояния диалога (т.е. какой текущий вопрос). Идей вообще нет, кроме как использовать NoSql Базу Данных.

**Comment by OneCellDM** [comment]
Использование базы данных наиболее удобный вариант) для хранения промежуточных данных и состояний.
Также вам поможет Машина состояний) 
Вариантов реализаций много, тут нужно понимать что вы хотите от бота. Если нужна помощь в реализации, то напишите мне, ссылки для связи есть в профиле, при детальном понимании задачи, можно накидать реализацию.
